chart/kubeapps/values.yaml

```diff
@@ -1632,6 +1632,12 @@ kubeappsapis:
             patch: 3
           ## @param kubeappsapis.pluginConfig.core.packages.v1alpha1.timeoutSeconds Value to wait for Kubernetes commands to complete
           timeoutSeconds: 300
+    kappController:
+      packages:
+        v1alpha1:
+          ## @param kubeappsapis.pluginConfig.kappController.packages.v1alpha1.upgradePolicy Default upgrade policy generating version constraints
+          ## enum: [ "major", "minor", "patch", "none" ]
+          upgradePolicy: patch
```

Comment:
Given that we'll (at some point) enable people to set the upgrade policy via the UX, can we name this `defaultUpgradePolicy`? (seems to be how it's being used below already anyway).

Reply:
Totally right! Naming is often the most difficult part. I've renamed it to`defaultUpgradePolicy`.